clang/test/Driver/sycl-device-optimizations.cpp

```diff
@@ -36,3 +36,14 @@
 // RUN:   | FileCheck -check-prefix=CHECK-DAE %s
 // CHECK-DAE: clang{{.*}} "-fenable-sycl-dae"
 // CHECK-DAE: sycl-post-link{{.*}} "-emit-param-info"
+
+/// Check that vectorizers are disabled by default:
+// RUN:   %clang -### -fsycl %s 2>&1 \
+// RUN:   | FileCheck -check-prefix=CHECK-VEC-DEFAULT %s
+// CHECK-VEC-DEFAULT-NOT: -vectorize-loops
+// CHECK-VEC-DEFAULT-NOT: -vectorize-slp
+/// Check that vectorizers can still be enabled manually:
+// RUN:   %clang -### -fsycl -fvectorize -fslp-vectorize %s 2>&1 \
+// RUN:   | FileCheck -check-prefix=CHECK-VEC-ENABLE %s
+// CHECK-VEC-ENABLE: -vectorize-loops
+// CHECK-VEC-ENABLE: -vectorize-slp
```

Comment:
(Not sure if the exact order is correct)
```suggestion
// CHECK-VEC-ENABLE: clang{{.*}} "-fsycl-is-device" "-vectorize-loops"{{.*} "-vectorize-slp"
```

Reply:
The order is correct, but a few regexps were missed/malformed